### ISSUE TITLE
Example Configuration Discussion

### ISSUE THREAD

**Issue by FynnBe** [bioimage-io/configuration_draft#1]
Let's gather and discuss...
(starting at 5fd8466)

here some input from my side to get going:
- move model name from `info: name` to `model: name`? 
- add hash for definition file?
- as long as we do not differenciate between x and y axes, etc, should we maybe just accept `d` (for dimension or `s` (spatial dim.) (and `c` obviously)?
- avoid too generic values like 'Normalize' (always state explicityly how?) -> `NormalizeMinMax`/`NormalizeMeanStd`
- do we really need to keep track of framework AND language (explicit vs. minimalistic...?)
- `training: pretrained_on:` should be a list?
- change `info` to `meta`? Everything is 'info'

**Comment by constantinpape** [comment]
> * move model name from `info: name` to `model: name`?

I thought about this too. Depends whether the name is metadata or actual part of the model definition.
We need to think about how to exactly ingest the model definition script (is there a defined entry point, can it be specified?). I will make a separate issue about this.
 
> * add hash for definition file?

Yes makes sense.

> * as long as we do not differenciate between x and y axes, etc, should we maybe just accept `d` (for dimension or `s` (spatial dim.) (and `c` obviously)?

Axis order might be relevant when dealing with different frameworks which can have different conventions also for spatial axes, so I would leave it in.

> * avoid too generic values like 'Normalize' (always state explicityly how?) -> `NormalizeMinMax`/`NormalizeMeanStd`

Sure, this is just a placeholder for whatever will be in the core library anyways.

> * do we really need to keep track of framework AND language (explicit vs. minimalistic...?)

Yes, because a framework can support multiple languages, e.g. pytorch supporting python and c++
or tensorflow python and java.

> * `training: pretrained_on:` should be a list?

Yes, makes sense.

> * change `info` to `meta`? Everything is 'info'

Makes sense.

**Comment by FynnBe** [comment]
- how about:
```yaml
data:
    output_type: logits # / probability
```
so we know where/if a sigmoid would be required

**Comment by constantinpape** [comment]
> how about:
> 
> ```yaml
> data:
>     output_type: logits # / probability
> ```
> 
> so we know where/if a sigmoid would be required

Or `output_transformations`, similar to `input_transformations`?

**Comment by constantinpape** [comment]
Also superseeded by hackathon discussions.
